src/com/facebook/buck/apple/AppleDescriptions.java

```diff
@@ -508,7 +509,8 @@ static AppleBundle createAppleBundle(
         Optional<FrameworkDependencies> frameworkDependencies =
             resolver.requireMetadata(
                 BuildTarget.builder(dep)
-                    .addFlavors(FRAMEWORK_FLAVOR)
+                    //this seems wrong
```

Comment:
Perhaps we can make a non-framework flavor and explicitly specify it in a rule's `deps` to include libraries that we want to force to be normal libs.  (Or analogously specify the framework flavor to force to be frameworks)